src/modules/dashboard/DashboardTopPanel.tsx

```diff
@@ -107,7 +108,16 @@ export const DashboardTopPanel = () => {
           )}
         </TopInfoPanelItem>
 
-        <TopInfoPanelItem icon={<NetAPYIcon />} title={<Trans>Net APY</Trans>} loading={loading}>
+        <TopInfoPanelItem
+          icon={<NetAPYIcon />}
+          title={
+            <div style={{ display: 'flex' }}>
```

Comment:
Not at the PC right now so little hard to check, but I think we have a predefined textwithtooltip component somewhere

Reply:
That component was for wrapping the actual tooltip text. There was another component which took text and tooltip as props called ListTopInfoItem but that was specific to the dashboard headers